lucida/calendar/Makefile

```diff
@@ -24,7 +24,7 @@ corenlp:
 	fi
 
 start_server: 
-	gnome-terminal -x bash -c "./gradlew run $(filter-out $@,$(MAKECMDGOALS)); read -n1"
+	./gradlew run
```

Comment:
I honestly don't know what these were doing. Every time I run with  `$(filter-out $@,$(MAKECMDGOALS)); read -n1` it says these commands are not found. 

According to the internet: `$(filter-out PATTERN...,TEXT)` returns all whitespace-separated words in `TEXT` that do not match any of the `PATTERN` words, removing the words that do match one or more. And `read -n1` means read one character at a time. But `MAKECMDGOALS` is not defined anywhere.